src/app/pages/AccountDetailsPage/hook.ts

```diff
@@ -50,3 +55,46 @@ export const useAccountTransactions = (scope: SearchScope, address: string) => {
     isTotalCountClipped,
   }
 }
+
+const WANTED_EVM_LOG_EVENTS: string[] = ['Transfer']
+
+export const useAccountTokenTransfers = (scope: SearchScope, address: string) => {
+  const { network, layer } = scope
+  const pagination = useSearchParamsPagination('page')
```

Comment:
(as usual, i dont like this in here)

Reply:
I agree, but at this point we are just repeating an established pattern. Let's refactor it in a separate PR at some point.